### ISSUE TITLE
DB issue with lowercase in SQL statement: Unable to fetch and process metadata

### ISSUE THREAD

**Issue by Stefan365** [syndesisio/syndesis#1853]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

Unable to fetch and process metadata, see steps to reproduce.
Errors are seen in steps 5 and 6. 
Error on web console:
```
ERROR TypeError: t.lineMachine is undefined
Stack trace:
4dAV/ui</t.prototype.updateFromConfig/<@https://syndesis.192.168.99.100.nip.io/main.29a32246d29c20888f96.bundle.js:1:398436
eFQL/</</</c</t.prototype.invokeTask@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:50302
onInvokeTask@https://syndesis.192.168.99.100.nip.io/main.29a32246d29c20888f96.bundle.js:1:1087546
eFQL/</</</c</t.prototype.invokeTask@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:50215
eFQL/</</</o</e.prototype.runTask@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:45472
eFQL/</</</a</e.invokeTask@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:51396
e/this.invoke<@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:51281
c/n.args[0]@https://syndesis.192.168.99.100.nip.io/polyfills.00b2490f9e8f9b4c7b5e.bundle.js:1:66337
 main.29a32246d29c20888f96.bundle.js:1:1070550

```
and following errors in meta.log (oc logs syndesis-meta-.... > meta.log)
[meta.log](https://github.com/syndesisio/syndesis/files/1785878/meta.log)

server.log and db.log looks OK.

This problem is somehow related only to CONTACT db table in the FINISH connection.
(not with TODO table. e.g. if in step 5 we specify eg. 
```
INSERT INTO TODO(task, completed) VALUES(:#task, 0), 
```
this misbehavior is not seen.)

## Screenshot

![db_to_db](https://user-images.githubusercontent.com/2714974/37047624-c9022f70-216b-11e8-9586-d7adf95fd658.png)

## API Endpoints and Schemas

POST https://syndesis.192.168.99.100.nip.io/api/v1/connections/5/actions/sql-connector
Request Body: 
{"query":"INSERT INTO CONTACT (first_name) values (:#fn)"}

response:
[sql-connector.json.txt](https://github.com/syndesisio/syndesis/files/1786002/sql-connector.json.txt)

## Tasks involved / Steps to Reproduce

1. Select Create Integration.
2. Select start connection PostgresDB.
3. Select action "Periodic SQL invocation" and specify query
    "SELECT * FROM TODO"
4. Select finish connection PostgresDB.
5. Select action "Ivoke SQL". and specify query related to CONTACT table
   "INSERT INTO CONTACT (first_name) values (:#fn)" 
6. add datamapper step

**Comment by gashcrumb** [comment]
Hmm, that data shape seems to be missing the specification fields in the response:

```
  "inputDataShape": {
    "kind": "json-schema"
  },
  "outputDataShape": {
    "kind": "json-schema"
  },
```

**Comment by zregvart** [comment]
I think this will be fixed with #1886

**Comment by KurtStam** [comment]
It looks like it's related to 'values' being lowercase in:

INSERT INTO CONTACT (first_name) values (:#fn)
